src/Command/Local/AbstractLocalCommand.php

```diff
@@ -63,4 +63,57 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
         $this->localHelper->setLogger($this->logger);
         $this->coreApi = $this->localHelper->api($this->environment);
     }
+
+    protected function uploadFile(string $filename): ?string
```

Comment:
Do not like this function here, because now all local commands have this function.
Maybe create a new parentCommand for FolderUploadCommand && UploadAssetsCommand.php. 
Or use a trait or maybe a service?

Reply:
Go for an abstract parent commend class: 8041d616e133421d05f67b4e1570ae5d09f1a63d